extensions/ql-vscode/src/vscode-tests/cli-integration/run-cli.test.ts

```diff
@@ -68,4 +70,16 @@ describe('Use cli', function() {
     const queryInfo: QueryInfoByLanguage = await cli.resolveQueryByLanguage(getOnDiskWorkspaceFolders(), Uri.file(queryPath));
     expect((Object.keys(queryInfo.byLanguage))[0]).to.eql('javascript');
   });
+
+  it.only('should resolve printAST queries', async function() {
```

Comment:
Should we make this loop over all languages? The filenames may differ.

Reply:
Good idea. I will change that.